test-cases/tests/backup-restore/j08-verify-user-sso-backup-and-restore.md

```diff
@@ -20,21 +20,45 @@ Note: this test should only be performed at a time it will not affect other ongo
 
 ### Postgres
 
-1. Verify Clients and Realms exist in postgres using the terminal in the `standard-auth` pod in the `redhat-rhmi-operator` namespace
+1. Verify Clients and Realms exist in postgres.
```

Comment:
would it make sense to add a note here that user should be logged into the cluster as kubeadmin using `oc`?

Reply:
Yeah makes, will update :+1: